.github/workflows/benchmarks.yml

```
@@ -0,0 +1,39 @@
+name: Node benchmarks
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10
```

I would use the last LTS node
```suggestion
        node-version: 12
```

---

Agreed, I used 10 because it was the LTS version when the PR was submitted.